### DISCUSSION TITLE
Please add a screenshot :)

### DISCUSSION THREAD

**Discussion by BernhardBaumrock** [adminerevo/adminerevo#126] [General]
Hi, I read this in the readme:

> but also to give Adminer a new features, layout, etc.

I've used adminer but never used adminerevo (yet), so as you are mentioning new layouts it would be nice to show this somewhere so that we can see how it looks without installing it somewhere.

I've also searched youtube for an intro about adminerevo but didn't find anything.

**Comment by BernhardBaumrock** [comment]
Update: [TracyDebugger](https://github.com/adrianbj/TracyDebugger) is using AdminerEvo now and I don't see any difference in the layout...

**Reply by adrianbj** [reply]
@BernhardBaumrock - I am running a custom theme in Tracy, so it doesn't look anything like the default anyway.

**Reply by BernhardBaumrock** [reply]
Ah, thx @adrianbj I wasn't aware of that! Yeah, so my request seems quite accurate then :)

**Reply by adrianbj** [reply]
Just in case you want to see what it looks like without that theme, you could just comment out:
https://github.com/adrianbj/TracyDebugger/blob/c9cb0670f039fb59b1608300b0b12aba322eef45/ProcessTracyAdminerRenderer.module.php#L55
